experimental/PropertyDDS/examples/partial-checkout/src/dataObject.ts

```diff
@@ -45,7 +45,9 @@ export class PropertyTree extends LazyLoadedDataObject<ISharedDirectory> impleme
         if (this.runtime.existing) {
             const treeHandle = await this.root.wait<IFluidHandle<SharedPropertyTree>>(propertyKey);
             if (this._queryString !== undefined) {
-                treeHandle.absolutePath += `?${  this._queryString}`;
+                // The absolutePath of the DDS should not be updated. Instead, a new handle can be created with the new
+                // path. To be fixed with this issue - https://github.com/microsoft/FluidFramework/issues/6036
+                (treeHandle as any).absolutePath += `?${  this._queryString}`;
```

Comment:
@DLehenbauer Created this issue to fix this - https://github.com/microsoft/FluidFramework/issues/6036

Reply:
I think we need a tool in the same release (same PR would be nice) giving people ability to create handles via handle + tail URI math. As same thing will happen in Bohemia repo and we need to have right answer (not just open an issue and add casting). I'd guess that 99% of them would be in the case of "add tail URI that does not participate in identifying GC node" and thus can be solved via providing single API to do this math

Reply:
That all said, this code should not do what it does, it should do .get() on original handle and follow up with request() on remaining IFluidRouter object - this is way more explicit then constructing handle. Maybe have a helper method for that too?

Reply:
No worries.  This was an expedient hack to plumb through extra information to a prototype service until we can surface the service properly.  It's not intended to last.